src/Page/Page.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { ThemeProvider } from '@emotion/react';
+import { StyledPage } from './pageStyles';
+import theme from 'src/styles/theme';
+
+export interface PageProps {
+    /**
+     * These are the classes that would be applied to the Page component.
+     */
+    className?: string;
+}
+
+/**
+ * Page Component
+ * 
+ * The Page Component can be used to render other child components.
+ */
+
+const Page: FC<PageProps> = props =>
```

Comment:
The indentation in this file seems different than other component files. Can you update to match?

Reply:
Done